test/benchmark/utils/parser.py

```diff
@@ -649,6 +653,20 @@ def write_compare_list_to_js(log_dir_str, case_information, output_filename):
         f.write(json_str)
     pass
 
+def copy_dir(src, dest):
+    try:
+        shutil.copytree(src, dest)
+    except shutil.Error as e:
+        print('Error generating html reports. Error: %s' % e)
```

Comment:
We may add a common error handling for parser, like throwing an error here to benchmark tests calling this function, and the test may output a log warning to the test result.